docs/09_firebase_deployment.md

```diff
@@ -0,0 +1,201 @@
+and go to the project setting (tap on the setting gear icon near the `Project Overview` on top of the left panel and select `Project settings`.
+2. Scroll down and find your Firebase Web Application. 
+3. Go to `Firebase SDK snippet` of your application, select `Config` and copy the generated code.
+4. Go to the `web/index.js` file in the application directory and replace this piece of code with the copied one in step 3: 
+    ```
+        var firebaseConfig = {
+          apiKey: "AIzaSyCkM-7WEAb9GGCjKQNChi5MD2pqrcRanzo",
+          authDomain: "metrics-d9c67.firebaseapp.com",
+          databaseURL: "https://metrics-d9c67.firebaseio.com",
+          projectId: "metrics-d9c67",
+          storageBucket: "metrics-d9c67.appspot.com",
+          messagingSenderId: "650500796855",
+          appId: "1:650500796855:web:65a4615a28f3d88e8bb832",
+          measurementId: "G-3DB4JFLKHQ"
+        };
+    ```
+
+Finally, you have a configured Flutter application that works with your Firebase instance.
+It's time to deploy your Flutter application to the Firebase Hosting!
+ 
+## Building and deploying the application to the Firebase Hosting
+
+### Preparing your environment 
+Before deploying metrics application, make sure you have the correct Flutter version installed,
+ by running the `flutter --version` command. You should have `v1.15.3` installed. 
+
+To switch the Flutter to the required version, you should run the `flutter version 1.15.3` command.
+
+Also, you should enable flutter web support by running the `flutter config --enable-web` command.
+
+### Building and deploying Flutter application
+ 
+1. Open the terminal and navigate to the metrics project folder.
+2. Run `firebase login` command and follow the instructions to log in to the Firebase CLI with your Google account.
+ Use the same account that you used to create your Firebase project (or the one that has access to it).
+3. After you have logged in, run the `firebase use --add` command, and select the project id of the project, created in previous steps.
+4. Give an alias to your project.
+5. Run the `flutter build web` command from the root of the metrics project to build the release version of the application.
+ It is recommended to add `--dart-define=FLUTTER_WEB_USE_SKIA=true` parameter to build the application with the `SKIA` renderer.
+4. Run the `firebase deploy --only hosting` command to deploy an application to the Firebase Hosting.
+
+After the deployment process finished, your application will be accessible using the `Hosting URL`, printed to console.
+
+## Creating test data for the deployed application
+
+Once you've deployed the metrics application to Firebase Hosting, you can create a sample data to test the application.  
+The application reads data from 2 Firestore collections: `projects` and `build`. To create a sample data follow the next steps: 
+
+1. Deploying the `seedData` cloud function: 
```

Comment:
Should we note that after seed data is created seedData function should be deactivated?